portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,36 +23,70 @@
 import java.util.List;
 import com.google.gson.Gson;
 import java.util.Arrays;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.util.*;
```

Comment:
Normally we don't import all classes in package. We should only import those we really need. By doing that, we could make the dependency more clear.

Reply:
Ok noted!